tests/e2e-scality/e2e-tests/README.md

```diff
@@ -0,0 +1,116 @@
+# End-to-End Tests for Scality S3 CSI Driver
+
+This directory contains end-to-end tests for the Scality S3 CSI Driver. The tests are written in Go and use the Kubernetes E2E framework with Ginkgo and Gomega for BDD-style testing.
+
+## Directory Structure
+
+- `kubernetes/` - Contains tests using the standard Kubernetes storage E2E framework
+- `scality/` - Contains Scality-specific tests for additional functionality
+
+## Running Tests Directly
+
+You can run the tests directly using the Go test command:
```

Comment:
This assumes that there is an S3 endpoint already available, right ?
Should we also include instruction for running cloudserver locally (you already have a `docker-compose` for that) ?

Reply:
You are right, along with that, I will add it makefile as well. We will need both S3 and CSI driver.
We will [remove all these tests](https://github.com/scality/mountpoint-s3-csi-driver/pull/13#discussion_r2046383803), these are just dummy tests. So I will rewrite a readme and S3 strategy for local testing will be done as well.
I made a note [here](https://scality.atlassian.net/browse/S3CSI-12).